package.json

```diff
@@ -43,30 +43,30 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.5.5",
-    "@rollup/plugin-replace": "^2.2.1",
-    "@types/jest": "^24.0.23",
-    "babel-eslint": "^10.0.3",
-    "babel-jest": "^24.9.0",
+    "@babel/core": "^7.8.6",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.8.6",
+    "@rollup/plugin-replace": "^2.3.1",
+    "@types/jest": "^25.1.3",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^25.1.0",
     "babel-plugin-dynamic-import-node": "^2.3.0",
-    "concurrently": "^5.0.0",
+    "concurrently": "^5.1.0",
     "custom-event": "^1.0.1",
-    "eslint": "6.3.0",
+    "eslint": "6.8.0",
     "eslint-config-canopy": "2.3.0",
     "eslint-config-important-stuff": "^1.1.0",
-    "husky": "^3.0.5",
-    "jest": "^24.9.0",
+    "husky": "^4.2.3",
+    "jest": "^25.1.0",
     "prettier": "^1.19.1",
     "pretty-quick": "^2.0.1",
-    "rimraf": "^3.0.0",
-    "rollup": "^1.20.3",
+    "rimraf": "^3.0.2",
+    "rollup": "^1.32.0",
```

Comment:
this one is the one I'm most scared of. I tried it out by using an override on a deployed application to my local single-spa and it was still working.